misc/mzutil/scripts/wait_for_view_states.py

```diff
@@ -109,19 +112,37 @@ def wait_for_materialize_views(args: argparse.Namespace) -> None:
     with psycopg2.connect(f"postgresql://{args.host}:{args.port}/materialize") as conn:
         while pending_views:
             views_to_remove = []
+            time_taken = time.time() - start_time
```

Comment:
it's probably worth using time.monotonic for this.

Reply:
Yes, that does seem preferable! I'll make the change, thanks!